graphql-spring-boot-test/src/main/java/com/graphql/spring/boot/test/GraphQLTestSubscription.java

```diff
@@ -207,6 +207,18 @@ public void reset() {
    * @return The received response.
    */
   public GraphQLResponse awaitAndGetNextResponse(final int timeout) {
```

Comment:
I wonder if the variants with the `int` parameter for `timeout` should be deprecated...